test/jdk/java/awt/dnd/InterJVMGetDropSuccessTest/InterJVMGetDropSuccessTest.java

```diff
@@ -275,19 +280,22 @@ public void run(String[] args) {
             }
 
             synchronized (Util.SYNC_LOCK) {
-                robot.mouseRelease(InputEvent.BUTTON1_MASK);
+                robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
                 Util.SYNC_LOCK.wait(Util.FRAME_ACTIVATION_TIMEOUT);
             }
 
-            if (!dragSourceListener.isDropFinished()) {
-                throw new RuntimeException("Drop not finished");
-            }
+            EventQueue.invokeAndWait(()-> {
```

Comment:
```suggestion
            EventQueue.invokeAndWait(() -> {
```
Added a space to follow the common code style.